zerver/lib/test_classes.py

```diff
@@ -2268,7 +2268,7 @@ def get_body(self, fixture_name: str) -> str:
         return body
 
 
-class MigrationsTestCase(ZulipTestCase):  # nocoverage
+class MigrationsTestCase(ZulipTransactionTestCase):  # nocoverage
```

Comment:
What's the story with this change?

It probably is a good idea, but it should probably be its own commit with a commit message.

Reply:
Here's the [discussion on CZO](https://chat.zulip.org/#narrow/stream/3-backend/topic/topic_wildcard_mentioned.20flag/near/1629148)